<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1037</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Comm Stats GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="4" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Left Side IR Sensor</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Bottom Left IR Sensor</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Right Side IR Sensor</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QTextBrowser" name="bottomleftBrowser"/>
        </item>
        <item row="1" column="3">
         <widget class="QTextBrowser" name="toprightBrowser"/>
        </item>
        <item row="3" column="3">
         <widget class="QTextBrowser" name="bottomrightBrowser"/>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Top Left IR Sensor</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Top Middle IR Sensor</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Top Right IR Sensor</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QTextBrowser" name="topmiddleBrowser"/>
        </item>
        <item row="1" column="1">
         <widget class="QTextBrowser" name="topleftBrowser">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QTextBrowser" name="bottommiddleBrowser"/>
        </item>
        <item row="2" column="3">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Bottom Right IR Sensor</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Bottom Middle IR Sensor</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QTextBrowser" name="leftsideBrowser"/>
        </item>
        <item row="5" column="2">
         <widget class="QTextBrowser" name="rightsideBrowser"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Server IP Address</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="serverIPAddressInput">
          <property name="text">
           <string>192.168.0.74</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Server Port</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="serverPortInput">
          <property name="text">
           <string>2000</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="connectToServer">
          <property name="text">
           <string>Connect to Server</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="sentCommStatLabel">
          <property name="text">
           <string>Sent commStat:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="requestButton">
          <property name="text">
           <string>Request Sensor Data</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
